@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -97,4 +97,24 @@ export default {
       reset: "Reset zoom",
     },
   },
+  DatePicker: {
+    close: "Close",
+    chooseDate: "Choose date",
+    chooseDates: "Choose dates",
+    chooseDateRange: "Choose start and end date",
+    chooseMonth: "Choose month",
+    week: "Week",
+    weekNumber: "Week {week}",
+    selectWeekNumber: "Select week {week}",
+    month: "Month",
+    nextMonth: "Go to next month",
+    previousMonth: "Go to previous month",
+    year: "Year",
+    nextYear: "Go to next year",
+    previousYear: "Go to previous year",
```

Comment:
Same as `goToNextMonth` comment further down, does it make sense to prefix these with `goTo` to show intention better?